package.json

```diff
@@ -8,7 +8,8 @@
   "license": "MIT",
   "dependencies": {
     "@expo/samples": "2.1.1",
-    "expo": "^30.0.1",
+    "expo": "32.0.6",
+    "metro-config": "^0.53.1",
     "react": "16.3.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
```

Comment:
Hey @lnmunhoz I think the react-native package would also need updating here.

Reply:
@Fried-Chicken I fixed the dependencies. The app seems to be running now on IOS.